### PR TITLE
Update branding to 2.1.27

### PR DESCRIPTION
**Pull request by wtgodbe**
Will update baseline on patch tuesday

### CODE REVIEW

**Comment by wtgodbe** [comment]
Test build: https://dev.azure.com/dnceng/internal/_build/results?buildId=1012346&view=results

**Review by JunTaoLuo — APPROVED** [review]
Looks good, but let's wait for verification of public and internal builds.

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -23,8 +23,14 @@ variables:
   - name: BuildScriptArgs
     value: ''
 - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+  - group: DotNet-MSRC-Storage
+
+  - name: _DOTNETCLIMSRC_READ_SAS_TOKEN
+    value: $(dotnetclimsrc-read-sas-token)
   - name: BuildScriptArgs
-    value: '/p:BuildNumber=$(Build.BuildId)'
+    value: '-AccessTokenSuffix $(_DOTNETCLIMSRC_READ_SAS_TOKEN)
```
Comment (dougbu) [inline .azure/pipelines/ci.yml:31]: Why not use `$(dotnetclimsrc-read-sas-token)` here❔

**Review by dougbu — APPROVED** [review]
One nit&hellip;

**Comment by wtgodbe** [comment]
> /Users/runner/work/1/s/src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj(57,5): error : Unable to find precompiled view file bin/Release/netstandard2.0/Microsoft.AspNetCore.Identity.UI.Razor.dll

I have no idea what this one's about...

File: .azure/pipelines/jobs/default-build.yml [hunk 2]
```diff
@@ -101,6 +101,9 @@ jobs:
       PB_AssetRootUrl: ''
       PB_RestoreSource: ''
       PB_PublishBlobFeedKey: ''
+      AccessTokenSuffix: ''
+    ${{ if ne(variables['System.TeamProject'], 'public') }}:
+      AccessTokenSuffix: '$(dotnetclimsrc-read-sas-token)'
```
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml:106]: Why are both `$(AccessTokenSuffix)` and `$(PB_ACCESSTOKENSUFFIX)` needed, especially containing the same value❔

**Comment by wtgodbe** [comment]
Now seeing a lot of test failures:

> Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher[8]
        Failed to invoke hub method 'Stream'.
  System.IO.InvalidDataException: Invocation provides 2 argument(s) but target expects 1.

>   Application startup exception: System.NotSupportedException: Duplicate definitions of 'OverloadedMethod'. Overloading is not supported.
     at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.DiscoverHubMethods() in /_/src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs:line 420

And others - Changing reference resolution patterns  may cause more work than we'd expected. @dougbu @JunTaoLuo how do you feel about treating 2.1 like 3.1/5.0 and waiting to update the baseline until patch tuesday?

File: .azure/pipelines/jobs/default-build.yml (hunk 2 above)
Comment (wtgodbe) [reply .azure/pipelines/jobs/default-build.yml:106]: I've been testing stuff out in internal builds so the .yml files are getting a bit ugly, I'll clean everything up once I get stuff working

**Comment by BrennanConroy** [comment]
Just FYI, those aren't test failures, they're expected exceptions during some tests. There are a ton of test failures in this PR though, those just aren't any of them.

**Comment by dougbu** [comment]
> how do you feel about treating 2.1 like 3.1/5.0 and waiting to update the baseline until patch tuesday?

This PR isn't actually updating the baselines for realz. The main difference is the increased use of project and non-baselined package references (more `$(UseProjectReferences)`, more `$(UseLatestPackageReferences)`). Perhaps the `$(UseLatestPackageReferences)` settings should be left alone e.g. if the previous baselines are actually _ahead_ of the checked-in versions.

**Comment by wtgodbe** [comment]
> This PR isn't actually updating the baselines for realz. The main difference is the increased use of project and non-baselined package references (more $(UseProjectReferences), more $(UseLatestPackageReferences)). Perhaps the $(UseLatestPackageReferences) settings should be left alone e.g. if the previous baselines are actually ahead of the checked-in versions.

Changing use of those switches is a result of having to update the baseline early - I'm wondering if there's a strong reason to try to get reference resolution working in an early-baseline world for 2.1, or if we can just leave reference resolution alone and delay the baseline updates until Patch Tuesday, like we're doing in 2.1/3.1. And, I ran the baseline generator against this branch, I thought it was updated "for real" (the only packages from last month were signalr javascript packages) - are there other updates you'd expect in the baseline?

**Comment by wtgodbe** [comment]
Just reverted the baseline & reference resolution changes here, the more I think about it the more it makes sense to just have the same behavior here as in 3.1/5.0 (update the baseline on patch tuesday). Saves us the headache of fixing reference resolution, checking in new feeds to sources.props every month, dealing with validating to internal feeds, and having 2 different sets of behavior across our servicing bands.

File: eng/Baseline.Designer.props [hunk 3]
```diff
@@ -1240,4 +1240,4 @@
     <BaselinePackageReference Include="Microsoft.AspNetCore.DataProtection.Extensions" Version="[2.1.1, )" />
     <BaselinePackageReference Include="Microsoft.Owin.Security" Version="[3.0.1, )" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```
Comment (dougbu) [inline eng/Baseline.Designer.props:1243]: Find a way to revert these changes or I bet CodeCheck will fail.

**Review by dougbu — APPROVED** [review]

File: eng/Baseline.Designer.props (hunk 3 above)
Comment (dougbu) [reply eng/Baseline.Designer.props]: Sorry, I should have mentioned the GitHub UI couldn't fix this
